x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/mobile/kpi_over_time_config.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ConfigProps, DataSeries } from '../../types';
+import { FieldLabels, OPERATION_COLUMN } from '../constants';
+import { buildPhraseFilter } from '../utils';
+import {
+  METRIC_SYSTEM_CPU_USAGE,
+  METRIC_SYSTEM_MEMORY_USAGE,
+  SERVICE_NAME,
+  TRANSACTION_DURATION,
+} from '../constants/elasticsearch_fieldnames';
+
+export function getMobileKPIConfig({ seriesId, indexPattern }: ConfigProps): DataSeries {
+  return {
+    id: seriesId,
+    reportType: 'service-latency',
+    defaultSeriesType: 'line',
+    seriesTypes: ['line', 'bar'],
+    xAxisColumn: {
+      sourceField: '@timestamp',
+    },
+    yAxisColumns: [
+      {
+        sourceField: 'business.kpi',
+        operationType: 'median',
+      },
+    ],
+    hasOperationType: true,
+    defaultFilters: [
+      'labels.net_connection_carrier_name',
+      'labels.device_model',
+      'labels.net_connection_type',
+      'host.os.platform',
+      'host.os.full',
+      'service.version',
+    ],
+    breakdowns: [
+      'labels.net_connection_carrier_name',
+      'labels.device_model',
+      'labels.net_connection_type',
+      'host.os.platform',
+      'host.os.full',
+      'service.version',
+      'labels.net_connection_carrier_isoCountryCode',
+    ],
+    filters: [...buildPhraseFilter('agent.name', 'iOS/swift', indexPattern)],
```

Comment:
we probably needs to discuss with APM-UI team on what other default filters we need to add here to fetch apm ios data.

Reply:
possibly also `open-telemetry/swift`